### ISSUE TITLE
Supports filed beforeUpdate

### ISSUE THREAD

**Issue by fundon** [balderdashy/sails-postgresql#163]
Sometimes, I have one filed and it uses `array` type.
When inserting/updating, I need to conver it to `{1,2,3,4}`, so I think that if it likes below code that will be good:

```js
attributes: {
  tags: {
    type: 'array',
    beforeUpdate: function () {
       attrs.tags = '{' + attrs.tags.toString() + '}';
    }
  },

 photos: {
    type: 'array',
    beforeUpdate: function (attrs) {
       attrs.photos = '{' + attrs.photos.toString() + '}';
    }
 }
}
```

I don't want to do this:

```js
attributes: {},

beforeUpdate: function (attrs, cb) {
  if (attrs.tags) {
   ....
  }
  if (attrs.photos) {
   ....
  }
  cb();
}


```

**Comment by sailsbot** [comment]
Thanks for posting, @fundon.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by fundon** [comment]
Why close this?
